src/platform/debugger/multiplexingDebugService.web.ts

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This seems too extreme to me. This code is a dupe of the other code. Can you not make a single file that just has the node specific parts optional?